### DISCUSSION TITLE
Hot reload is not working for frontend?

### DISCUSSION THREAD

**Discussion by sid0-0** [ToolJet/ToolJet#9212] [Q&A]
Currently I'm having to restart local server after each change T-T.
Is this some issue with my local setup or is it a known issue.

I tried to fix it as well, but can't find the source of this issue. Webpack config seems to be set up correctly. But sourcemaps are also not produced. Any help on where I should look for the fix?

**Comment by ShazanRizvi** [comment]
Hello @sid0-0,
It can be an issue with your local setup as it seems the server is not rebuilt as soon as the changes are done. Can you please share the guide that you have followed for setup?
For implementation-related questions, our Slack community is recommended as our engineers are active there. Here is the invite: https://tooljet.com/slack.
